src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton/BitToggleButton.razor.cs

```diff
@@ -14,6 +14,9 @@ public partial class BitToggleButton
         /// <summary>
         /// Whether the toggle button can have focus in disabled mode.
         /// </summary>
+        /// <defaultValue>
+        /// true
```

Comment:
Shall we instantiate components to find out what are properties default values? @msynk

Reply:
honestly, I couldn't understand what you mean clearly

Reply:
If we instantiate components (for example BitToggleButton), we can find out what are its properties' default values!
I'd like to know you and @msynk opinion about this idea

Reply:
I think its a good suggestion

Reply:
So we can use Activator.CreateInstance and use that instance to find out what's the default value of each property.

Reply:
What do you think about using "[System.ComponentModel.DefaultValue]" annotation above our properties which we want to have default values instead of that? 

I can also use the way you suggested but I think we will have trouble with reference type properties and it always returns null  , unless we assign value to them when they are creating, but I'm not really sure about it

I must say that I can use both scenarios together, at first I can check the attribute and if it isn't found then I use the second way to know its default value.

Let me know your opinions. 
Thanks in advance